### PR TITLE
interface: allows reading sd cards internal info from `block-devices` interface

### PR DESCRIPTION
**Pull request by sergey-borovkov**
Without this line:
	cat /sys/block/mmcblk0/device/date
results in this denial:
/sys/devices/platform/soc/3f202000.mmc/mmc_host/mmc0/mmc0:aaaa/date

See: https://forum.snapcraft.io/t/interface-for-read-info-about-sd-card/25202

After adding the line in the commit to my snap apparmor file it works:

root@srly-r7d8mg4oj4eo0go:/home/pi# sudo apparmor_parser -r /var/lib/snapd/apparmor/profiles/snap.screenly-client.submit-report 
Warning from /var/lib/snapd/apparmor/profiles/snap.screenly-client.submit-report (/var/lib/snapd/apparmor/profiles/snap.screenly-client.submit-report line 1297): Character # was quoted unnecessarily, dropped preceding quote ('\') character
root@srly-r7d8mg4oj4eo0go:/home/pi# snap run --shell screenly-client.submit-report
root@srly-r7d8mg4oj4eo0go:/home/pi# cat /sys/block/mmcblk0/device/date 
06/2017
Thanks for helping us make a better snapd!
Have you signed the [license agreement](https://www.ubuntu.com/legal/contributors) and read the [contribution guide](https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md)?
yes

### CODE REVIEW

**Review by mardy — APPROVED** [review]
Looks good to me, but it would be nice to have a review from security team as well.

**Review by anonymouse64 — APPROVED** [review]
lgtm, but yes would be good to have security review for this

**Review by alexmurray — APPROVED** [review]
LGTM.

**Comment by mvo5** [comment]
Should we cherry-pick this for 2.51 so that it's available in stable very soon ? @sergey-borovkov ?

**Comment by sergey-borovkov** [comment]
> Should we cherry-pick this for 2.51 so that it's available in stable very soon ? @sergey-borovkov ?

That would be great, thank you.

**Comment by mvo5** [comment]
@sergey-borovkov Thank you! This is done now.

**Pull request merged**
